c/src/ml-api-common-tizen-feature-check.c

```diff
@@ -97,25 +100,27 @@ _ml_tizen_get_feature_enabled (ml_feature_e ml_feature)
   ml_tizen_initialize_feature_state ();
 
   g_mutex_lock (&feature_info->mutex);
-  feature_enabled = feature_info->feature_state;
+  feature_enabled = feature_info->feature_state[ml_feature];
   g_mutex_unlock (&feature_info->mutex);
 
   if (NOT_SUPPORTED == feature_enabled) {
-    _ml_loge ("machine_learning.inference NOT supported");
+    _ml_loge ("Tizen machine learning feature (%s) is NOT supported.",
+        ML_FEATURES[ml_feature]);
     return ML_ERROR_NOT_SUPPORTED;
```

Comment:
Minor: How about using `_ml_error_report_return`?

Reply:
Internally we use the macro check_feature_state() and this will set error message in ml-error-report.